### PR TITLE
add attributes to request tab

### LINKED ISSUE #714: Add Request attributes to the Request tab
It would be useful to allow exploring the request attributes from the Request panel, specially now we are going to have a lot of useful data related to authentication, like the user identity.

### PR DESCRIPTION
**Pull request by saeideng**
close #714

### CODE REVIEW

**Comment by saeideng** [comment]
this will duplicate the `params` key
`params` shows in `Routing Params` section and in this section

File: templates/element/request_panel.php [hunk 1]
```diff
@@ -34,6 +34,9 @@
 <h4><?= __d('debug_kit', 'Routing Params') ?></h4>
 <?= $this->Toolbar->dump($params) ?>
 
+<h4><?= __d('debug_kit', 'Attributes') ?></h4>
+<?= $this->Toolbar->dump($attributes); ?>
```
Comment (markstory) [inline templates/element/request_panel.php]: Should we remove `Routing Params` instead of having duplicate data?

Comment (saeideng) [reply templates/element/request_panel.php]: not sure how users feel about `Routing Params` , and they know it is in `attributes` list or no

Comment (markstory) [reply templates/element/request_panel.php]: That's fair, but if it is the only params list in the panel they will figure it out. We will be leaking some implementation details, but those details are already a public interface. The `getParam()` methods are ease of use methods as the required existence checks to safely use the return of `getAttribute()` are tedious.

Comment (saeideng) [reply templates/element/request_panel.php]: removed

Comment (markstory) [reply templates/element/request_panel.php]: We should have an isset check around `$attributes` as history records in the debug kit DB may not have attributes set.

Comment (saeideng) [reply templates/element/request_panel.php:35]: `getAttributes()` always have at least 4 items :point_down:  

https://github.com/cakephp/cakephp/blob/41ce80827fd964f6b32e799c837fc3c585a097c1/src/Http/ServerRequest.php#L1674-L1679

Comment (saeideng) [reply templates/element/request_panel.php]: added

**Pull request merged**